villains-only/src/app/services/user.service.ts

```diff
@@ -70,19 +71,18 @@ export class UserService {
             .catch(this.handleError);
     }
 
-    public getHeroByEmail(user: User): Observable<User> {
-        const body = JSON.stringify(user);
-        const headers = new Headers({ 'Content-Type': 'application/json' });
-        const options: RequestOptions = new RequestOptions({ headers: headers });
-
-        return this.http
-            .post(`http://localhost:8090/VillainsOnly/getUserByEmail.app`, body, options)
-            .map((response: Response) => {
-                return <User>response.json();
-            })
-            .catch(this.handleError);
-
-    }
+    // public getHeroByEmail(user: User): Observable<User> {
+    //     const body = JSON.stringify(user);
+    //     const headers = new Headers({ 'Content-Type': 'application/json' });
+    //     const options: RequestOptions = new RequestOptions({ headers: headers });
+
+    //     return this.http
```

Comment:
We can just take this out instead of just commenting it out.

Reply:
u rite